### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jxah**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,8 +3,8 @@
...
-author        |editor
---------------|-------
-jxah          |escambron
---------------|--------------------
-read/edit/help|read/edit/help/corect
+|author          |editor               |
+|----------------|---------------------|
+|jxah            |escambron            |
+|----------------|---------------------|
+|read/edit/help  |read/edit/help/corect|
```
Comment (jxah) [inline README.md]: some more coming

File: README.md [hunk 2]
```diff
@@ -3,8 +3,8 @@
 ********************************************************************************************************************************************************************
 hope you enjoy
 
-author        |editor
---------------|-------
-jxah          |escambron
---------------|--------------------
-read/edit/help|read/edit/help/corect
```
Comment (jxah) [inline README.md:10]: goodbye

File: README.md [hunk 3]
```diff
@@ -3,8 +3,8 @@
 ********************************************************************************************************************************************************************
 hope you enjoy
```
Comment (jxah) [inline README.md:4]: for sure

File: README.md (hunk 2 above)
Comment (jxah) [reply README.md:10]: not even to you @escambron.

**Review by maparce — APPROVED** [review]
me gusta

**Review by jxah — COMMENTED** [review]
dunDUNyay

**Pull request merged**